build.bat Failing

I designed my launcher exactly as the instructions say, and run build.bat. Build.bat gets a Windows SmartScreen error, which I had to fix. When i got Build.bat to run, it went smoothly until this: "C:\Users\TrTha\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher.sln" (Build target) (1) ->
       "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher-Appx\DistroLauncher-Appx.vcxproj.metaproj" (default target) (3)
        ->
       "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher-Appx\DistroLauncher-Appx.vcxproj" (default target) (4) ->
       (CustomBuild target) -> 
         C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC
       \VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited
        with code 1. [C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLa
       uncher-master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj]

This is my entire run:
Microsoft (R) Build Engine version 15.7.179.6572 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 7/24/2018 8:30:41 PM.
     1>Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher
       -master\DistroLauncher.sln" on node 1 (Build target(s)).
     1>ValidateSolutionConfiguration:
         Building solution configuration "Debug|x64".
     1>Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher
       -master\DistroLauncher.sln" (1) is building "C:\Users\\Downloads\WSL-Dist
       roLauncher-master\WSL-DistroLauncher-master\DistroLauncher\DistroLauncher.vcxp
       roj" (2) on node 1 (default targets).
     2>InitializeBuildStatus:
         Creating "x64\Debug\launcher.tlog\unsuccessfulbuild" because "AlwaysCreate" 
         was specified.
       CustomBuild:
         All outputs are up-to-date.
       ClCompile:
         All outputs are up-to-date.
         All outputs are up-to-date.
       ResourceCompile:
         All outputs are up-to-date.
       Link:
         All outputs are up-to-date.
         DistroLauncher.vcxproj -> C:\Users\\Downloads\WSL-DistroLauncher-master
         \WSL-DistroLauncher-master\x64\Debug\launcher.exe
       FinalizeBuildStatus:
         Deleting file "x64\Debug\launcher.tlog\unsuccessfulbuild".
         Touching "x64\Debug\launcher.tlog\launcher.lastbuildstate".
     2>Done Building Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-
       DistroLauncher-master\DistroLauncher\DistroLauncher.vcxproj" (default targets)
       .
     1>Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher
       -master\DistroLauncher.sln" (1) is building "C:\Users\\Downloads\WSL-Dist
       roLauncher-master\WSL-DistroLauncher-master\DistroLauncher-Appx\DistroLauncher
       -Appx.vcxproj.metaproj" (3) on node 1 (default targets).
     3>Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher
       -master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj.metaproj" (3) is build
       ing "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-mas
       ter\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj" (4) on node 1 (default ta
       rgets).
     4>InitializeBuildStatus:
         Touching "x64\Debug\Ubuntu14.04.tlog\unsuccessfulbuild".
       CustomBuild:
         Copy C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-m
         aster\\x64\Debug\launcher.exe into C:\Users\\Downloads\WSL-DistroLaunch
         er-master\WSL-DistroLauncher-master\\x64\Debug\Ubuntu14.04\ubuntu14.04.exe
         The system cannot find the path specified.
                 0 file(s) copied.
     4>C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\V
       CTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited w
       ith code 1. [C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLaun
       cher-master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj]
     4>Done Building Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-
       DistroLauncher-master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj" (defaul
       t targets) -- FAILED.
     3>Done Building Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-
       DistroLauncher-master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj.metaproj
       " (default targets) -- FAILED.
     1>Done Building Project "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-
       DistroLauncher-master\DistroLauncher.sln" (Build target(s)) -- FAILED.

Build FAILED.

       "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher.sln" (Build target) (1) ->
       "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher-Appx\DistroLauncher-Appx.vcxproj.metaproj" (default target) (3)
        ->
       "C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLauncher-master\
       DistroLauncher-Appx\DistroLauncher-Appx.vcxproj" (default target) (4) ->
       (CustomBuild target) -> 
         C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC
       \VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited
        with code 1. [C:\Users\\Downloads\WSL-DistroLauncher-master\WSL-DistroLa
       uncher-master\DistroLauncher-Appx\DistroLauncher-Appx.vcxproj]

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:40.05

I have a certificate file, install.tar.gz of Ubuntu Base 14.04.5 (http://cdimage.ubuntu.com/ubuntu-base/releases/14.04/release/) amd64 and everything the guide tells me to do.

From what I understand, something went wrong on a build command.
Thank You.

--- a/README.md
+++ b/README.md
@@ -69,7 +69,7 @@ The distro launcher is comprised of two Visual Studio projects - `launcher` and 
 ``` xml
 <PropertyGroup Label="Globals">
   ...
-  <ProjectName>mydistro</ProjectName>
+  <TargetName>mydistro</TargetName>
 </PropertyGroup>
 ```
 
@@ -77,7 +77,7 @@ So, if I wanted to instead call my distro "TheBestDistroEver", I'd change this t
 ``` xml
 <PropertyGroup Label="Globals">
   ...
-  <ProjectName>TheBestDistroEver</ProjectName>
+  <TargetName>TheBestDistroEver</TargetName>
 </PropertyGroup>
 ```
 

